boring/src/sizing/code generator/format_inputs.py

```diff
@@ -0,0 +1,131 @@
+# -*- coding: utf-8 -*-
```

Comment:
I think this is very creative. Justin (the OpenMDAO development lead) recommends using Sympy or JAX instead of wolfram, but I'm assuming you chose wolfram over sympy for a reason?